app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass

```

Comment:
It's a good idea to provide a custom error message for the `VaccineError` class. This will make your error handling more descriptive and easier to understand. You can do this by overriding the `__init__` method in your class. Here's an example:

python
class VaccineError(Exception):
    def __init__(self, message="Vaccine error occurred."):
        super().__init__(message)
```